app/models/state_file_nj_intake.rb

```diff
@@ -150,10 +153,32 @@ def disqualifying_df_data_reason
     return :exempt_interest_exceeds_10k if tax_exempt_interest_income > 10_000
   end
 
+  def eligibility_claimed_as_dependent?
+    if self.filing_status_mfj?
+      self.direct_file_data.claimed_as_dependent? && self.direct_file_data.spouse_is_a_dependent?
+    else
+      self.direct_file_data.claimed_as_dependent?
+    end
+  end
+
+  def eligibility_made_less_than_threshold?
+    nj_gross_income = calculator.lines[:NJ1040_LINE_29].value
+    threshold = self.filing_status_single? || self.filing_status_mfs? ? 10_000 : 20_000
+    nj_gross_income <= threshold
+  end
+
+  def health_insurance_eligibility
+    if self.eligibility_all_members_health_insurance_no?
+      has_exception = self.eligibility_made_less_than_threshold? || self.eligibility_claimed_as_dependent?
+      has_exception ? "eligible" : "ineligible"
+    else
+      "eligible"
+    end
+  end
+
   def disqualifying_eligibility_rules
     {
-      eligibility_lived_in_state: "no",
-      eligibility_out_of_state_income: "yes",
```

Comment:
[dust, could be boulder] why were these two rules removed here?

Reply:
to bring this full circle -- i believe the explanation given in-person was that these two rules were not actually having an effect so could be safely removed. @aloverso please correct me if i'm misunderstanding